Auth0/JWTAlgorithm.swift

```diff
@@ -22,10 +18,16 @@ enum JWTAlgorithm: String {
             !signature.isEmpty else { return false }
         switch self {
         case .rs256:
-            guard let publicKey = jwk.rsaPublicKey, let rsa = A0RSA(key: publicKey) else { return false }
-            let sha256 = A0SHA()
-            return rsa.verify(sha256.hash(data), signature: signature)
+            var verificationError: Unmanaged<CFError>?
```

Comment:
No need for passing an error instance, if we're not going to use it. Just pass `nil` to the respective parameter of `SecKeyVerifySignature` and remove the error variable.